src/calendar/stories/Calendar.stories.tsx

```diff
@@ -63,3 +64,38 @@ const Template: Story = () => {
 
 export const Default = Template.bind({});
 Default.args = {};
+
+const RangeTemplate: Story = () => {
+  return (
+    <RangeCalendar>
```

Comment:
Users just need to do this, they get normal/Range

```suggestion
    <Calendar>
```